include/fpm/ios.hpp

```diff
@@ -737,4 +737,44 @@ std::basic_istream<CharT, Traits>& operator>>(std::basic_istream<CharT, Traits>&
 
 }
 
+#if __cplusplus >= 202002L
+#   include <version>
+#endif
+#ifdef __cpp_lib_format
+#   include <format>
+#   include <sstream>
+template<typename CharT, typename B, typename I, unsigned int F, bool R>
+struct std::formatter<fpm::fixed<B, I, F, R>, CharT>
+{
+    bool showpos = false;
+
+    template<class ParseContext>
+    constexpr typename ParseContext::iterator parse(ParseContext& ctx)
+    {
+        auto it = ctx.begin();
+        if(it != ctx.end() && *it == '+')
+        {
+            showpos = true;
+            ++it;
+        }
+        if(it != ctx.end() && *it != '}')
+        {
+            throw std::format_error("Invalid format");
+        }
+        return it;
+    }
+
+    template<typename FormatContext>
+    typename FormatContext::iterator format(const fpm::fixed<B, I, F, R>& value, FormatContext& ctx) const
+    {
+        std::basic_ostringstream<CharT> out;
+        if(showpos)
+            out << std::showpos;
+        out << value;
+
+        return std::ranges::copy(std::move(out).str(), ctx.out()).out;
```

Comment:
Since this doesn't assume that `std::format` exists (as per the version check), should this assume that `std::ranges` exists? Perhaps a more standard `std::copy` is preferable here.